partcad-cli/src/partcad_cli/click/command.py

```
@@ -0,0 +1,193 @@
+
+
+@cli.result_callback()
+def process_result(result, verbose, quiet, no_ansi, package, format):
+    # TODO-89: @alexanderilyin: What is this for?
+    if not no_ansi:
+        pc.logging_ansi_terminal_fini()
+
+    # Abort if there was at least one error reported during the exeution time.
+    # `result` is needed for the case when the command was not correct.
+    if pc.logging.had_errors or result:
+        raise click.Abort()
```

Ruff (0.8.2)

Unnecessary parentheses on raised exception

Remove unnecessary parentheses

(RSE102)